nexus/tests/output/nexus_tags.txt

```diff
@@ -0,0 +1,81 @@
+hardware
```

Comment:
@davepacheco like this:

```suggestion
operations for tag "hardware"
```

Reply:
That is what I meant.  Even more explicit though:

```suggestion
API operations found with tag "hardware"
OPERATION NAME                          URL PATH
```